apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -2,12 +2,18 @@
  * Generated by `createservice property.RegisterResidentService --type mutations`
  */
 
+const omit = require('lodash/omit')
+
 const { getById, GQLCustomSchema } = require('@core/keystone/schema')
-const access = require('../access/RegisterResidentService')
-const { Resident } = require('../utils/serverSchema/index')
-const { Property } = require('@condo/domains/property/utils/serverSchema')
-const { Resident: ResidentAPI } = require('../utils/serverSchema')
 
+const { Property: PropertyAPI } = require('@condo/domains/property/utils/serverSchema')
+const { getAddressUpToBuildingFrom } = require('@condo/domains/property/utils/serverSchema/helpers')
+
+const { Resident: ResidentAPI } = require('@condo/domains/resident/utils/serverSchema')
+const access = require('@condo/domains/resident/access/RegisterResidentService')
+
+// these fields won't be changed while updating Resident
+const SKIP_RESIDENT_FIELDS = ['address', 'addressMeta', 'unitName']
```

Comment:
you do not need to take out constants if they impair the readability of the code.

Reply:
to understand the business logic of the RegisterResidentService, it is necessary to see which fields are skipped

Reply:
This constant will not be used anywhere except in one place. And in such cases, it is better to inline the value

Reply:
fixed